src/aind_hcr_data_transformation/compress/czi_to_zarr_s3.py

```diff
@@ -93,7 +93,7 @@ def create_spec(
             "fill_value": 0,
             "chunk_grid": {
                 "name": "regular",
-                "configuration": {"chunk_shape": shard_shape},
+                "configuration": {"chunk_shape": chunk_shape},
```

Comment:
Should I leave this as shard shape and change line 385 to be dataset.chunk_layout.read_chunk.shape ?